src/runtime/types/form.ts

```diff
@@ -36,7 +36,7 @@ export type InferOutput<Schema> = Schema extends StandardSchemaV1 ? StandardSche
 
 export type FormData<S extends FormSchema, T extends boolean = true> = T extends true ? InferOutput<S> : InferInput<S>
 
-export type FormInputEvents = 'input' | 'blur' | 'change' | 'focus'
+export type FormInputEvents = 'touched' | 'blur' | 'dirty'
```

Comment:
This feels like a regression in terms of feature. Is there a way to restore previous validation triggers with Formwerk? 

Also this type is used for the `validateOn` prop on the Form component, I don't think `touched` and `dirty` fit in that category.